### PR TITLE
Update hi.js

### PR DESCRIPTION
**Pull request by ThomasShelby69**
IGN - akshutcoder

### CODE REVIEW

File: hi.js [hunk 1]
```diff
@@ -912,14 +912,14 @@ chall.name.airk Get {0} mid-air kills {1}
...
 chall.name.drk Get {0} drift kills {1}
-chall.name.kstk Get a {0} killstreak {1}
+chall.name.kstk Get a {0} किल स्ट्रीक {1}
 chall.name.w Win {0} मैचेस
 chall.name.d Die {0} समय
 chall.name.score Get {0} अंक
 chall.name.dmg Deal {0} डैमेज {1}
 chall.name.bzk Get {0} बज़ किलज़ {1}
 chall.name.cc Get {0} close-calls {1}
-chall.name.rk Deal {0} revenge kills {1}
```
Comment (TheDuce) [inline hi.js:922]: let this be revenge kill itself

File: hi.js [hunk 2]
```diff
@@ -510,7 +510,7 @@ windows.controls.aim1 एम्
 windows.controls.shoot1 गोली चलाना
 windows.controls.inspect इंस्पेक्ट
 windows.controls.spray स्प्रे
-windows.controls.sprayWheel Spray Wheel (Hold)
+windows.controls.sprayWheel स्प्रे पाइयाँ (दबाएं)
```
Comment (TheDuce) [inline hi.js:513]: `स्प्रे पाइयाँ (Hold)` works?

File: hi.js (hunk 1 above)
Comment (ThomasShelby69) [reply hi.js:922]: Alright

File: hi.js (hunk 2 above)
Comment (ThomasShelby69) [reply hi.js:513]: Hmm, I'll change that.

**Review by TheDuce — APPROVED** [review]

**Comment by mty22** [comment]
covered in https://github.com/mty22/krunker.io-translations/pull/458

**Pull request closed without merging**